code/modules/events/spacevine.dm

```diff
@@ -466,9 +466,10 @@
 		return
 	if(parent)
 		SV.mutations |= parent.mutations
-		var/parentcolor = parent.atom_colours[FIXED_COLOUR_PRIORITY]
-		SV.add_atom_colour(parentcolor, FIXED_COLOUR_PRIORITY)
-		if(prob(mutativness))
+		if(parent.atom_colours)
```

Comment:
>By the way, why arent we defining the list directly, instead of in New()?

Because doing any call for the default value of a var (either list(), new(), icon(), etc) adds a hidden proc to that item that is ran before New() to actually init that, this hidden proc incurs the proc call overhead. Ie, adding such a call to /atom/ would slow down world initialization by a few seconds.